src/scss/_root.scss

```diff
@@ -0,0 +1,9 @@
+:root {
+  --color-dark-teal: #196076;
+  --color-light-peach: #fff6f6;
+  // --color-light-peach: rgba(243, 103, 72, 0.08);
+  --color-bright-coral: #F36748;
```

Comment:
Maybe consider converting these to `hsl`? Nothing technically wrong with hex codes, but other color formats tend to be easier to work with.

Reply:
I updated ✅ I also noticed I had that custom property on line 4 still commented out (I hadn't known what to do with it), and figured out where it was supposed to be used. So that got updated too. https://github.com/marissahuysentruyt/marissahuysentruyt/pull/21/commits/df993e2a9741b4fc0569b91d96a9c058e71ccee4